### PR TITLE
Update docs: allowing GitLab SSH URLs to be used on pack download

### PR DESCRIPTION
**Pull request by asthLucas**
Update docs/source/packs.rst

Fix sphinx-autobuild host argument

Set the Ubuntu base image to a version where Python 3.6 is the default

### CODE REVIEW

File: docs/source/packs.rst [hunk 1]
```diff
@@ -133,6 +133,10 @@ version or **upgrade to latest** if the version is not specified. Your config fi
 overwritten, so you can revert to an older version just as easily, but for production deployments
 we recommend to always specify versions in case there are major changes in ``latest``.
 
+.. note::
+
+    It is also possible to download and install packs from GitLab using a SSH based URL through the web interface in the Actions section.
```
Comment (amanda11) [inline docs/source/packs.rst:138]: Suggestion that add "New feature! As of StackStorm 3.4." to the beginning of the note, like the other notes where features are only available in certain releases.

**Review by amanda11 — CHANGES_REQUESTED** [review]

**Review by amanda11 — APPROVED** [review]
Looks ok to me, but as I made a minor change to wording would like approval from someone else as well.

File: docs/source/packs.rst [hunk 2]
```diff
@@ -133,6 +133,10 @@ version or **upgrade to latest** if the version is not specified. Your config fi
 overwritten, so you can revert to an older version just as easily, but for production deployments
 we recommend to always specify versions in case there are major changes in ``latest``.
 
+.. note::
+
+    New feature! As of StackStorm 3.4. It is also possible to download and install packs from GitLab using a SSH based URL through the web interface in the Actions section.
```
Comment (armab) [inline docs/source/packs.rst:138]: Looks like the section promises a new feature installing pack from gitlab server via Web UI only, while it's just a new format for pack install.

The documentation refers to https://github.com/StackStorm/st2/pull/5050 change and I think it just worth providing an example with `gitlab@` syntax somewhere in the "Installing a Pack" section above.

Comment (amanda11) [reply docs/source/packs.rst]: @armab @asthLucas I've updated where the gitlab url is mentioned in the doc based on review comment. 
I've based the example on what is in https://github.com/StackStorm/st2/pull/5050 PR. 
However I have to say that I have been using the git@ successfully with gitlab, so I'm not sure what scenarios require gitlab@.

File: Makefile [hunk 3]
```diff
@@ -72,7 +72,7 @@ livedocs: docs .livedocs
 	@echo "                       RUNNING DOCS"
 	@echo "==========================================================="
 	@echo
-	. $(VIRTUALENV_DIR)/bin/activate; sphinx-autobuild -H 0.0.0.0 -b html $(DOC_SOURCE_DIR) $(DOC_BUILD_DIR)/html
+	. $(VIRTUALENV_DIR)/bin/activate; sphinx-autobuild --host 0.0.0.0 -b html $(DOC_SOURCE_DIR) $(DOC_BUILD_DIR)/html
```
Comment (blag) [inline Makefile:75]: :+1:

**Review by blag — APPROVED** [review]

**Comment by blag** [comment]
LGTM

File: docs/source/packs.rst [hunk 4]
```diff
@@ -99,8 +99,14 @@ Exchange, and you can install your own packs from git just as easily.
...
 
-    # Install your own pack from git
+    # Install your own pack from git using http(s)
     st2 pack install https://github.com/emedvedev/chatops_tutorial
+    
+    # Install your own pack from git using ssh
+    st2 pack install git@github.com/emedvedev/chatops_tutorial
+    
+    # Install your own pack using gitlab URL (added in release 3.4)
+    st2 pack install gitlab@gitlab.com:example/examplepack
```
Comment (armab) [inline docs/source/packs.rst:109]: :100:

**Pull request merged**